docs/4.3/admin-guide.md

```diff
@@ -2259,11 +2261,58 @@ When upgrading multiple clusters:
 
 ## Backing Up Teleport
 
+When planning a backup of Teleport, it's important to know what is where and the
+importance of each component. Teleports Proxies and Nodes are stateless, and thus
+only `teleport.yaml` should be backed up.
+
+The Auth server is Teleports brains, and depending on the backend should be backup
+in a regular cadance.
+
+For example a customer running Teleport on AWS with etd have these key items of data:
+
+| What | Where ( Example AWS Customer ) |
+|-|-|
+| Local Users ( not SSO )  | DynamoDB |
+| Certificate Authorities | DynamoDB |
+| Trusted Clusters | DynamoDB |
+| Connectors: SSO | DynamoDB / File System  |
+| RBAC | DynamoDB / File System |
+| teleport.yaml | File System |
+| teleport.service | File System |
+| license.pem | File System |
+| TLS /. | ( File System / Outside Scope )  |
+
+For this customer, we would Recommend using [AWS best practices](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/BackupRestore.html) for backing up DynamoDB. If DynamoDB is used for
+the audit log, logged events have a TTL of 1 year.
+
+| Backend | Recommend Backup Strategy  |
+|-|-|
+| dir ( local filesystem )   | Copy `data_dir/storage` and use `tctl get all` to get local state. |
+| DynamoDB | [Follow AWS Guidelines for Backup & Restore](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/BackupRestore.html) |
+| etcD | [Follow etcD Guidleines for Disaster Recovery ](https://etcd.io/docs/v2/admin_guide) |
+| Firestore | [Follow GCP Guidlines for Automated Backups](https://firebase.google.com/docs/database/backups) |
+
+### Config Files
+
+We would recommend you store configs as discrete files in git.
+
+### GitOps
+
+If running Teleport at scale it's important for teams to have an automated way to
+restore Teleport. At a high level, this is our recommend approach:
```

Comment:
I wouldn't classify this as backup/restore.
This is more about applying configs by committing files to a repo.